pkg/apis/networking/v1alpha1/types.go

```diff
@@ -14,29 +14,39 @@ type LoadBalancerList struct {
 
 // +genclient=true
 
-// LoadBalancer describe a LoadBalancer witch providers Load balancing for services
+// LoadBalancer describes a LoadBalancer which provides Load Balancing for applications
+// LoadBalancer contains a proxy and multiple providers
```

Comment:
Highlight internal and external property of lb

LoadBalancer contains a proxy and multiple providers to load balance either internal or external traffic.